index.js

```diff
@@ -138,19 +138,21 @@ function transformMessage(data) {
 
     /** Prepare message for TeslaMate */
     let power = "";
-    let isCharging = false;
-    if (associativeArray["Gear"]) {
```

Comment:
By checking we received a `Gear` it means that the power is left empty, in the old streaming API the power is set to `0` if not driving, since it is not implemented yet my suggestion is to hardcode power to `0` as well and not letting it flip flop between `0` and `""`